osu.Game/Screens/Play/Player.cs

```diff
@@ -236,7 +238,7 @@ public void Resume()
         {
             lastPauseActionTime = Time.Current;
             hudOverlay.KeyCounter.IsCounting = true;
-            hudOverlay.Progress.Hide();
+            hudOverlay.HideProgress();

```

Comment:
this is going to break if we ever allow replays to be paused (i'm pretty sure this will happen).